cogs/induct.py

```diff
@@ -193,12 +200,17 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
             )
             return
 
-        if induction_member.bot:
-            await self.command_send_error(
-                ctx,
-                message="Member cannot be inducted because they are a bot.",
-            )
-            return
+        intro_channel: discord.TextChannel | None = discord.utils.get(
+            self.bot.main_guild.text_channels, name="introductions",
+        )
+
+        if intro_channel:
+            for m in await intro_channel.history(limit=30).flatten():
+                if m.author.id == induction_member.id:
+                    await m.add_reaction(":wave:")
+                    await m.add_reaction(":TeX:")
+                    continue
```

Comment:
I think the intended keyword is `break`. This will exit the loop after reacting to the most recent message.

The `continue` keyword will just move on to the next message so could result in multiple reactions. The `continue` keyword is also unnecessary here as the end of the current loop will already be reached without the `continue` statement.